app/containers/GamePage/index.js

```diff
@@ -75,13 +94,46 @@ export default class GamePage extends Component {
     });
   }
 
+  loadParticipants() {
+    const { database, participants } = this.state;
+    const { params } = this.props;
+    const usersRef = database.ref(`${params.rid}/participants`);
+    usersRef.off();
+    usersRef.limitToLast(12).on('child_added', (data) => {
+      const val = data.val();
+      participants.push({ key: data.key, name: val.name, role: '', health: 5, hand: [] });
+      this.setState({ participants });
+      usersRef.once('value', (snapshot) => {
```

Comment:
Are you sure yo're supposed to do this? I don't recommend putting a event listener into another listener just to get the data